src/main/scala/beam/router/ParquetRequester.scala

```diff
@@ -83,7 +85,7 @@ object ParquetRequester extends BeamHelper with LazyLogging {
 
     val requests = requestRecords.map { req =>
       val reqJsonStr = new String(req.get("requestAsJson").asInstanceOf[Utf8].getBytes, StandardCharsets.UTF_8)
-      io.circe.parser.parse(reqJsonStr).right.get.as[RoutingRequest].right.get
+      io.circe.parser.parse(reqJsonStr).getOrElse(throw new NoSuchElementException("Either.right.get on Left")).as[RoutingRequest].getOrElse(throw new NoSuchElementException("Either.right.get on Left"))
```

Comment:
`get` is deprecated. I opted to keep the old implementation of throwing to use a left

Reply:
An alternative would be to eat the exception and do a match, returning empty for a left?